tasks/internal/push-artifacts-to-cdn-task/push-artifacts-to-cdn-task.yaml

```diff
@@ -952,11 +1038,14 @@ spec:
         # shell check wants us to find ./* but that adds `./` to the paths which breaks the script
         # shellcheck disable=SC2035
         while IFS= read -r -d '' file ; do
+            # disabling debug here because it extrapolates the Termination Message Log size and the result is not
```

Comment:
But I don't see any disabling here? Also, this issue was fixed by `grep -v "^\+"`, right? Maybe this comment is an omission? And the `set -x` afterwards is not needed either?

Reply:
same case as above. I added it while in a previous fix and left it there after I could fix it in a better way. I will remove it as well.